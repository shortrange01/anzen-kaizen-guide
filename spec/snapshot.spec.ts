import path from 'path';

describe('スナップショットテスト', () => {
    beforeEach(async () => {
        await page.goto(`file://${path.resolve(__dirname, '../index.html')}`);
    });

    it('スナップショットテスト', async () => {
        await page.waitForSelector('#title', { visible: true });
        expect(await page.content()).toMatchSnapshot();
    });
});
