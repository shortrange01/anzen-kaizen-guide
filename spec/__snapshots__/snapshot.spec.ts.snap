// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`スナップショットテスト スナップショットテスト 1`] = `
"<!DOCTYPE html><html><head>
        <meta charset=\\"utf-8\\">
        <link rel=\\"stylesheet\\" href=\\"./css/style.css\\">
        <title>TODO</title>
        <style>
            h1 {
                font-size: 14px;
            }
        </style>
    </head>
    <body>
        <h1 id=\\"title\\">ページタイトル</h1>
        <button id=\\"addTodo\\">タスクを追加する</button>
        <div id=\\"todoList\\" style=\\"display: none;\\"></div>
        <div id=\\"todoEmpty\\">タスクがありません</div>
        <div>
            <span id=\\"nextTodo\\">次のTODO: (未登録)</span>
            <span id=\\"todoCount\\">(全0件)</span>
        </div>
        <script src=\\"./dist/main.js\\"></script>
    

</body></html>"
`;
